front/app/components/ProjectAndFolderCards/components/Topbar/show.ts

```diff
@@ -0,0 +1,31 @@
+import { IStatusCounts } from 'hooks/useAdminPublicationsStatusCounts';
+import { isNilOrError, NilOrError } from 'utils/helperUtils';
+import { ITopicData } from 'services/topics';
+import { IAreaData } from 'services/areas';
+
+export const getShowFilters = (
```

Comment:
Let's make those named arguments with an object? They are too many and just impossible to keep track of and know what they are otherwise.

Reply:
:+1: